packages/shared-components/src/components/single-tooltip/single-tooltip.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, State, h } from '@stencil/core';
+
+@Component({
+  tag: 'single-tooltip',
+  styleUrl: 'single-tooltip.scss',
+  shadow: true,
+})
+export class Tooltip {
```

Comment:
Of course we need to wrap the tippy tooltip inside here.

Reply:
This might help in the re-positioning https://atomiks.github.io/tippyjs/v6/all-props/#getreferenceclientrect